app/presenters/publishing_api/html_attachment_presenter.rb

```diff
@@ -56,12 +56,20 @@ def base_path
     end
 
     def details
-      {
+      details_hash = {
         body: body,
         public_timestamp: public_timestamp,
         first_published_version: first_published_version?,
         political: political?,
       }
+
+      maybe_add_national_applicability(details_hash)
+    end
+
+    def maybe_add_national_applicability(content)
```

Comment:
It might be nice to name this parameter `details_hash` rather than `content`, for the sake of consistency.

```suggestion
    def maybe_add_national_applicability(details_hash)
```